config.py

```diff
@@ -442,3 +474,8 @@ def use_devnet_config(self, paths: OPPaths):
         self.sub_safety_margin = 4
 
 ####################################################################################################
+
+L2_EXECUTION_DATA_DIR = "db/L2-execution"
+"""Directory to store the op-geth blockchain data."""
```

Comment:
Ultimately this should be inlined into the `__init__` directly, but I supposed this still exists for the sake of the `clean` function?

Reply:
Yep, it's for the `clean` function? I replaced the call in `start_l2_execution_node` with `config.data_dir`.